ffmeta/data/ffmetadata_ddl.sql

```diff
@@ -60,122 +60,131 @@ CREATE TABLE IF NOT EXISTS `raw2` (
   `wave` char(30) CHARACTER SET latin1 DEFAULT NULL,
   `section` char(1) DEFAULT NULL,
   `leaf` char(30) DEFAULT NULL,
+  `obs` int(4) DEFAULT NULL,
+  `min` int(7) DEFAULT NULL,
```

Comment:
Not sure why `min` and `max` have different display widths. If these are small integers then perhaps `4` should suffice for both.